### PR TITLE
Bugfix/1299/finance/invoice download with invoice number

### PR DESCRIPTION
**Pull request by abhishekkhanduri**
#{1299}

### Description
Invoice File download with invoice number in new tab.

### CODE REVIEW

File: Modules/Invoice/Resources/views/edit.blade.php [hunk 1]
```diff
@@ -1,6 +1,9 @@
 @extends('invoice::layouts.master')
 @section('content')
-
+@php
+$invoice_file_path=$invoice->file_path;
+$filename=basename($invoice_file_path);
```
Comment (P4NK4J) [inline Modules/Invoice/Resources/views/edit.blade.php:5]: How about `$filename=basename($invoice->file_path);` ?

File: package-lock.json [hunk 2]
```diff
@@ -1,16983 +1,6 @@
 {
-    "name": "portal",
```
Comment (P4NK4J) [inline package-lock.json:2]: Why we are removing this file ?

**Review by P4NK4J — CHANGES_REQUESTED** [review]
Left some comments, please take a look

**Comment by P4NK4J** [comment]
@abhishekkhanduri There are some comments left to resolve in your PR, please check and then assign for review. Also, post a response corresponding to the comments.

File: Modules/Invoice/Resources/views/edit.blade.php (hunk 1 above)
Comment (abhishekkhanduri) [reply Modules/Invoice/Resources/views/edit.blade.php:5]: Yes this would be perfect.

File: package-lock.json (hunk 2 above)
Comment (abhishekkhanduri) [reply package-lock.json:2]: I have not made any changes in this file on my own.

File: .husky/pre-commit [hunk 3]
```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
```
[the remark this answers is not in the document]
Comment (abhishekkhanduri) [reply .husky/pre-commit]: This file will be created automatically when we runs npm install command.

File: package-lock.json (hunk 2 above)
Comment (abhishekkhanduri) [reply package-lock.json:2]: This file will be created automatically when we run install composer.

**Review by P4NK4J — APPROVED** [review]

**Pull request merged**